server/routes/farcast.ts

```diff
@@ -83,6 +85,9 @@ router.post('/callback', authMessage, getPageSetting, async (req, res): Promise<
   // TODO: mint stuff
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const { owner } = await lookupFid(fid)
+  const tokenData = ethers.utils.hexlify(ethers.utils.toUtf8Bytes('0'))
+  const tx = await mint(fid, 1, tokenData)
```

Comment:
I would add to a promise queue with capacity 1, and return immediately, so it doesn't block